### PR TITLE
Allow  'multiple' cardinality for scoring CustomOperator

### PR DESCRIPTION
**Pull request by zagovorichev**

### CODE REVIEW

**Comment by zagovorichev** [comment]
@krampstudio  maybe version increase was too much?

File: scripts/update/Updater.php [hunk 1]
```diff
@@ -318,6 +318,7 @@ public function update($initialVersion){
         }
 
         $this->skip('2.20.0', '2.20.1');
+        $this->skip('2.20.1', '2.21.0');
```
Comment (krampstudio) [inline scripts/update/Updater.php]: only one skip is sufficient : 
```php
$this->skip('2.20.0', '2.20.2');
```

File: manifest.php [hunk 2]
```diff
@@ -26,7 +26,7 @@
     'label' => 'QTI item model',
 	'description' => 'TAO QTI item model',
     'license' => 'GPL-2.0',
-    'version' => '2.20.1',
+    'version' => '2.21.0',
```
Comment (krampstudio) [inline manifest.php]: you've not added a new feature but fixed one, so the version should be `2.20.2`

**Comment by zagovorichev** [comment]
@krampstudio update

**Pull request merged**